src/components/ha-code-editor.ts

```diff
@@ -152,10 +153,8 @@ export class HaCodeEditor extends ReactiveElement {
         saveKeyBinding,
       ] as KeyBinding[]),
       this._loadedCodeMirror.langCompartment.of(this._mode),
-      this._loadedCodeMirror.theme,
-      this._loadedCodeMirror.Prec.fallback(
```

Comment:
It's not clear to me why the precedence for the syntax highlighting was lowered.  If this was to account for a bug, there was some fixes for highlighting precedence in the release notes, but I don't know what to check for in HA.  I made the assumption it was not needed anymore for now.